mlflow/utils/rest_utils.py

```diff
@@ -77,6 +77,13 @@ def http_request_safe(host_creds, endpoint, **kwargs):
     Wrapper around ``http_request`` that also verifies that the request succeeds with code 200.
     """
     response = http_request(host_creds=host_creds, endpoint=endpoint, **kwargs)
+    return http_response_handler(response, endpoint)
+
+
+def http_response_handler(response, endpoint):
```

Comment:
Suggestion (nit): Call this something like verify_rest_response? To me, response handler suggests that this is actually gonna process the result.

Reply:
makes sense